### PR TITLE
Alha dev

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Review by oliver-walker-hms — DISMISSED** [review]

File: src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+/**
+ * Class for accessing (reading and writing) files on the Ewon Flexy filesystem.
```
Comment (TomKimsey) [inline src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java]: Maybe rephrase this comment. Since file access is done in a standard way if you were just accessing the file you wouldn't need this. They are more of file access helpers/utils.

**Review by TomKimsey — DISMISSED** [review]
Approved but you may want to consider my comment.

File: src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java (hunk 1 above)
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java]: changed

**Review by TomKimsey — APPROVED** [review]

**Review by oliver-walker-hms — APPROVED** [review]
